src/themes/default/globals/reset.overrides

```diff
@@ -62,7 +62,7 @@ hr {
  */
 
 pre {
-    font-family: monospace, monospace; /* 1 */
+    font-family: monospace; /* 1 */
```

Comment:
just a typo, or is specifying font family twice fixes something?

Reply:
Explained here https://github.com/necolas/normalize.css/issues/519#issuecomment-197131966 . So it's intention